### PR TITLE
Drop some -Wmaybe-unitialized workarounds to help coverity

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

**Review by bluca — APPROVED** [review]
lgtm

File: src/network/networkd-routing-policy-rule.c [hunk 1]
```diff
@@ -790,7 +790,7 @@ int link_request_static_routing_policy_rules(Link *link) {
 }
 
 int request_process_routing_policy_rule(Request *req) {
-        RoutingPolicyRule *ret = NULL;  /* avoid false maybe-uninitialized warning */
+        RoutingPolicyRule *ret = NULL;
```
Comment (poettering) [inline src/network/networkd-routing-policy-rule.c]: did you intend to remove the `= NULL` here too?

Comment (keszybz) [reply src/network/networkd-routing-policy-rule.c]: I meant to, but gcc warns when that is done. I dropped that chunk now.

**Comment by poettering** [comment]
lgtm

**Pull request merged**